modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/service/ModelService.java

```diff
@@ -28,9 +28,9 @@
 
     List<ModelDto> getModels(String metricKey);
 
-    void addModelParams(ModelParams modelParams);
+    void addModelParams(String modelUUID, Hyperparams hyperparams);
 
-    void markToRebuild(String modelUUID, String metricKey, Boolean toRebuild);
+    void markToRebuild(String modelUUID, String metricKey, boolean toRebuild);
 
     void updateThresholds(String modelUUID, String metricKey, Map<String, Object> thresholds);
```

Comment:
I believe there are only two thresholds here: `weakSigmas` and `strongSigmas`. (I like "sigmas" here because it makes it explicit what sort of threshold we are talking about--it's not the constant thresholds.) If this is right then we don't want a general string-to-object map.